liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/diffChangelog.test.groovy

```diff
@@ -37,11 +37,11 @@ Optional Args:
     Default: null
   driverPropertiesFile (String) The JDBC driver properties file
     Default: null
-  excludeObjects (String) Objects to exclude from diff
+  excludeObjects (String) Regular expression of columns to exclude. Defaults to null.
     Default: null
   includeCatalog (Boolean) If true, the catalog will be included in generated changeSets. Defaults to false.
     Default: false
-  includeObjects (String) Objects to include in diff
+  includeObjects (String) Regular expression of columns to include. Defaults to null.
```

Comment:
+1

Reply:
I changed the wording to "Objects to include in diff. Supports regular expressions. Defaults to null."